content/en/docs/installation/system-requirements.md

```diff
@@ -0,0 +1,44 @@
+---
+title: System Requirements for Armory Enterprise
+linkTitle: System Requirements
+description: >
+  Hardware and other requirements to run Armory Enterprise
+---
+
+## Browsers
+
+You can use any modern browser to interact with the UI.
```

Comment:
Do we want to put a limit on `modern` or is it generally understood what that means?